Description: Fix diagnostics about exceeding partition limits
 Use the correct value in the diagnostics that print the upper bound on
 partition size and largest starting sector number.  Before, each would use
 UINT32_MAX.  Now each prints the partition-table-specific limit.
Origin: upstream, http://git.debian.org/?p=parted/parted.git;a=commitdiff;h=0fb5838f39c2c16ec72775e00702689f4b84f5c1
Forwarded: yes
Last-Update: 2010-04-21

Index: b/libparted/labels/pt-tools.c
===================================================================
--- a/libparted/labels/pt-tools.c
+++ b/libparted/labels/pt-tools.c
@@ -127,7 +127,7 @@
 			     " of %jd"),
 			   part->geom.length,
 			   pt_type,
-			   UINT32_MAX);
+			   pt_lim->max_length);
       return 0;
     }
 
@@ -140,7 +140,7 @@
 			   " of %jd"),
 			 part->geom.start,
 			 pt_type,
-			 UINT32_MAX);
+			 pt_lim->max_start_sector);
     return 0;
   }
 
